src/common/config.go

```diff
@@ -55,6 +55,10 @@ type Config struct {
 	// which OCI implementation to use for the docker sandbox (e.g., runc or runsc)
 	Docker_runtime string `json:"docker_runtime"`
 
+	// The max lambda timeout given in milliseconds
+	// If no timeout is given by the lambda, this max timeout is also the default
+	Lambda_Max_timeout int64 `json:"lambda_max_timeout"`
```

Comment:
1. let's put it in LimitsConfig
2. don't need to prefix with "lambda_" (or we would prefix nearly everything like that)
3. let's suffix with "_ms" so it's obvious what the units are (like we suffix with "_mb" below)

Reply:
Sure we can do that! :+1: 

(by the way I probably won't make much further changes until the weekend since I've got to work on a ECE 554 project that's due in a couple days) I might be able to slip this in tomorrow (hopefully) after we're done with the "mini-project")